client/packages/reports/src/ListView/StockAndItemWidget.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  Grid,
+  BarIcon,
+  Widget,
+  Typography,
+  ChevronDownIcon,
+} from '@openmsupply-client/common';
+import { useTranslation } from '@common/intl';
+import { ReportRowFragment } from '@openmsupply-client/system';
+
+interface StockReportProps {
+  reports: ReportRowFragment[] | undefined;
+}
+
+export const StockAndItemWidget: React.FC<StockReportProps> = ({ reports }) => {
+  const t = useTranslation('reports');
+
+  return (
+    <>
+      <Widget title={t('heading.stock-and-items')} Icon={BarIcon}>
+        <Grid
+          container
+          justifyContent="flex-start"
+          flex={1}
+          flexDirection="column"
+          paddingTop={2}
+        >
+          {reports?.map(report => (
+            <Grid
+              sx={{
+                display: 'flex',
+              }}
+              key={report?.id}
+            >
+              <Typography
+                variant="h6"
+                sx={{
+                  paddingBottom: 2,
+                }}
+              >
+                {report?.name}
```

Comment:
=/ the report name would have to be descriptive enough for the user to understand what it does =/

Reply:
Yep, it would need to be descriptive, we also might have name and "display name" (for translation), and maybe some extra info to describe the report on the report records